Dataset_Metrics/Dataset_metrics.ipynb

```diff
@@ -0,0 +1,2564 @@
+{
```

Comment:
Would mind stripping the binary and text output before merging? 

It will make future changes easier to review and merge.

Reply:
We should have pre-commit hooks for this btw; we'll figure that out when we move to gerrit.